src/App.tsx

```diff
@@ -17,38 +68,35 @@ export const App: React.FC = () => {
             type="text"
             placeholder="Enter a part of the name"
             className="input"
+            value={handleValue()}
+            onChange={handlerQuery}
+            onFocus={() => setIsFocused(true)}
+            onBlur={() => setIsFocused(true)}
           />
         </div>
-
         <div className="dropdown-menu" role="menu">
-          <div className="dropdown-content">
-            <div className="dropdown-item">
-              <p className="has-text-link">Pieter Haverbeke</p>
-            </div>
-
-            <div className="dropdown-item">
-              <p className="has-text-link">Pieter Bernard Haverbeke</p>
-            </div>
-
-            <div className="dropdown-item">
-              <p className="has-text-link">Pieter Antone Haverbeke</p>
-            </div>
-
-            <div className="dropdown-item">
-              <p className="has-text-danger">Elisabeth Haverbeke</p>
-            </div>
-
-            <div className="dropdown-item">
-              <p className="has-text-link">Pieter de Decker</p>
-            </div>
-
-            <div className="dropdown-item">
-              <p className="has-text-danger">Petronella de Decker</p>
-            </div>
-
-            <div className="dropdown-item">
-              <p className="has-text-danger">Elisabeth Hercke</p>
-            </div>
+          <div
+            className="dropdown-content"
+          >
+            {(isFocused && !query)
+            && (!selectedPerson) && peopleFromServer.map(person => (
+              <List
+                person={person}
+                onSelected={onSelected}
+                key={person.name}
+              />
+            ))}
+            {(query && !selectedPerson) && filteredPeople.map((person) => (
+              <List
+                person={person}
+                onSelected={onSelected}
+                key={person.name}
+              />
+            ))}
+            {(!selectedPerson && !filteredPeople.find(
+              p => p.name.includes(query),
+            ) && query)
+              && 'No matching suggestions'}
```

Comment:
something wrong with conditions because I see this unnecessary white space:
![image](https://github.com/mate-academy/react_autocomplete/assets/102289654/3088bb18-0f93-4520-a48a-5a590f9724c9)

Reply:
conditions should be refactored 100%